src/react-components/src/components/Link/Link.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import { uid } from "react-uid";
+import { LinkTypes, LinkIconPositions } from "./LinkTypes";
+import { element } from "prop-types";
+import Icon from "../Icons/Icon";
+import { iconRotationTypes } from "../Icons/IconTypes";
+
+export interface LinkProps {
+    /** Controls the link visuals—action, button, or default. */
+    linkType?: LinkTypes;
+
+    /** Href attribute */
+    href?: string;
+
+    /** BEM specifiers */
+    modifiers?: string[];
+
+    /** BEM specifiers */
+    blockName?: string;
+
+    /** Additional attributes, such as rel=nofollow, to pass to the <a> tag */
+    attributes?: {};
+}
+
+export default function Link(props: React.PropsWithChildren<LinkProps>) {
+    let { href, modifiers, blockName, attributes } = props;
+
+    let link_base_class = "link";
+
+    let childProps = {};
+
+    let icon;
+
+    let elementChildren = React.Children.map(
+        props.children,
+        (child: React.ReactElement) => {
+            if (typeof child === "string" && !props.href) {
+                throw new Error("Link needs prop 'href'");
+            } else if (child.type === "a") {
+                return child.props.children;
+            } else {
+                return child;
+            }
+        }
+    );
+
+    if (
+        props.linkType === LinkTypes.Action ||
```

Comment:
Is there a fallback for if linkType doesn't get passed? (You can add it as `linkType={LinkTypes.Action}` in L27